underactuated.html

```diff
@@ -1312,20 +1329,65 @@ <h1>The Simple Pendulum</h1>
 
   <section><h1>Exercises</h1>
 
-    <exercise><h1></h1>
+    <exercise><h1>Graphical Analysis</h1>
+
+      <figure>
+      <img width="60%" data-src="figures/exercises/graphical_analysis.svg"/>
+      <figcaption>First-order systems with multiple equilibria.</figcaption>
+      </figure>
+
+      Consider the first-order system $$\dot x = \begin{cases} - x^5 + 2 x^3 - x & \text{if} \quad x \leq 1, \\ 0 & \text{if} \quad 1 < x \leq 2, \\ - x + 2 & \text{if} \quad x > 2, \end{cases}$$ whose dynamics is represented in the figure above.  Notice that, together with $x^*=-1$ and $x^*=0$, all the points in the closed interval $[1,2]$ are equilibria for this system.  For each equilibrium point, determine whether it is stable i.s.L., asymptotically stable, and/or exponentially stable.
+
+    </exercise>
+
+    <exercise><h1>Basin of Attraction and Bifurcation Diagram</h1>
 
       Consider the first-order system with polynomial dynamics $$\dot x = f(x) = - x^3 + 4x^2 +11x - 30.$$
 
       <ol type="a">
 
         <li> Sketch the graph of the function $f(x)$, and identify all the equilibrium points.</li>
```

Comment:
maybe "sketch or plot"?  it's not meant to be an exercise in sketching polynomials...  alternatively, we could just plot it for them here?